### ISSUE TITLE
Mystery object does not appear on Acceleration screen

### ISSUE THREAD

**Issue by matthew-blackman** [phetsims/forces-and-motion-basics#302]
Discussing lesson use-cases of this sim with @kathy-phet, we noticed that the mystery box appears on the 'Friction' screen but not the 'Acceleration' screen. We agreed that it would be good to add it to the 'Acceleration' screen so that students can use the tools in that screen to solve for the mass of the box.

**Comment by matthew-blackman** [comment]
This is implemented and ready for review @kathy-phet, how would you like me to proceed?

https://github.com/phetsims/forces-and-motion-basics/assets/108756567/cb6a8981-f5ad-452b-9b42-963dc4c8405d

**Comment by kathy-phet** [comment]
I thought we were just making an issue - so it would be there when we visit the sim next. I didn't intend to implement it.

**Comment by amanda-phet** [comment]
This could likely be addressed with https://github.com/phetsims/forces-and-motion-basics/issues/304

**Comment by AgustinVallejo** [comment]
I reimplemented @matthew-blackman initial pair of commits for this issue. Please review if this is the intended behavior and close @marlitas

**Comment by amanda-phet** [comment]
Discussed 1/18/24 and would like the mystery object added, but don't need the trash can (since the water bucket is 100kg).

**Comment by Luisav1** [comment]
Removed the trash can object in the above commit.
![image](https://github.com/phetsims/forces-and-motion-basics/assets/52978048/a4984b5b-565e-4fa3-ac68-eb4fcb3ab35f)
Closing.

**Comment by Luisav1** [comment]
@amanda-phet caught that the trashcan was removed in the motion screen too. 
![image](https://github.com/phetsims/forces-and-motion-basics/assets/52978048/05f2558d-33ce-4b4a-96f5-50ecdf3c6982)
Reopening to keep the trash can on the other screens except the last one.

**Comment by amanda-phet** [comment]
It's missing from both Motion and Friction.

**Comment by Luisav1** [comment]
It seems I overlooked that the MotionModel is utilized for all three screens: Motion, Friction, and Acceleration. I've addressed this by conditionally adding back the trash can based on the screen in the commit above.

Here's how the Motion and Friction screens look now:
![image](https://github.com/phetsims/forces-and-motion-basics/assets/52978048/fd62db1f-4301-4d85-83a4-e273e41e448c)
![image](https://github.com/phetsims/forces-and-motion-basics/assets/52978048/8bd41e1c-7dbe-494d-b4d5-8e8032d29523)
Closing.
